litmus-portal/frontend/src/components/Sections/Settings/AccountsTab/AccountSettings/index.tsx

```diff
@@ -55,9 +45,20 @@ const AccountSettings: React.FC = () => {
     err: false,
   });
 
+  // handleCurrPassword handles password validation for first password field
+  const handleCurrPassword = (prop: keyof Password) => (
+    event: React.ChangeEvent<{ value: string }>
+  ) => {
+    setCurrPassword({
+      ...currPassword,
+      err: false,
```

Comment:
You don't need `err` the validation can handle it. You can remove `err` from all the password local states